segfault in CI

```
==================================== ERRORS ====================================
_______ ERROR at setup of test_tasklist_skip_taskbar_defaults[1-wayland] _______

request = <SubRequest 'tasklist_manager' for <Function test_tasklist_skip_taskbar_defaults[1-wayland]>>
manager_nospawn = <test.helpers.TestManager object at 0x7f348e3f5490>
override_xdg = False
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f348e43edd0>
position = 'top'

    @pytest.fixture
    def tasklist_manager(request, manager_nospawn, override_xdg, monkeypatch, position):
        monkeypatch.setattr("libqtile.widget.tasklist.has_xdg", override_xdg)
    
        config = getattr(request, "param", dict())
    
        class TasklistConfig(Config):
            auto_fullscreen = True
            groups = [
                libqtile.config.ScratchPad(
                    "SCRATCHPAD",
                    dropdowns=[
                        libqtile.config.DropDown("dd-a", spawn_cmd("dd-a"), on_focus_lost_hide=False),
                    ],
                ),
                libqtile.config.Group("a"),
                libqtile.config.Group("b"),
            ]
            layouts = [layout.Stack()]
            floating_layout = libqtile.resources.default_config.floating_layout
            keys = []
            mouse = []
            screens = [
                Screen(**{position: bar.Bar([TaskListTestWidget(name="tasklist", **config)], 28)})
            ]
    
>       manager_nospawn.start(TasklistConfig)

test/widgets/test_tasklist.py:69: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test/helpers.py:232: in start
    error = rpipe.recv()
            ^^^^^^^^^^^^
/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/multiprocessing/connection.py:250: in recv
    buf = self._recv_bytes()
          ^^^^^^^^^^^^^^^^^^
/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/multiprocessing/connection.py:430: in _recv_bytes
    buf = self._recv(4)
          ^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <multiprocessing.connection.Connection object at 0x7f3491ffecf0>
size = 4, read = <built-in function read>

    def _recv(self, size, read=_read):
        buf = io.BytesIO()
        handle = self._handle
        remaining = size
        while remaining > 0:
            chunk = read(handle, remaining)
            n = len(chunk)
            if n == 0:
                if remaining == size:
>                   raise EOFError
E                   EOFError

/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/multiprocessing/connection.py:399: EOFError
---------------------------- Captured stdout setup -----------------------------
2026-03-27 13:52:31,287 libqtile core.py:log_cb():L105  [backend/backend.c:342] Loading user-specified backends due to WLR_BACKENDS: headless
2026-03-27 13:52:31,288 libqtile core.py:log_cb():L105  [backend/headless/backend.c:60] Creating headless backend
2026-03-27 13:52:31,288 libqtile core.py:log_cb():L105  [util/env.c:25] Loading WLR_RENDERER option: pixman
2026-03-27 13:52:31,288 libqtile core.py:log_cb():L105  [render/pixman/renderer.c:328] Creating pixman renderer
2026-03-27 13:52:31,289 libqtile core.py:log_cb():L105  [backend/headless/backend.c:17] Starting headless backend
2026-03-27 13:52:31,290 libqtile core.py:log_cb():L105  [server.c:190] Updating: 0,0 (800x600)
2026-03-27 13:52:31,297 libqtile core.py:log_cb():L105  [server.c:116] Running Wayland compositor on WAYLAND_DISPLAY=wayland-0
---------------------------- Captured stderr setup -----------------------------
Fatal Python error: Segmentation fault

Thread 0x00007f3495485b80 (most recent call first):
  File "/home/runner/work/qtile/qtile/libqtile/backend/wayland/core.py", line 332 in on_config_load
  File "/home/runner/work/qtile/qtile/libqtile/core/manager.py", line 174 in load_config
  File "/home/runner/work/qtile/qtile/libqtile/core/manager.py", line 233 in async_loop
  File "/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/asyncio/events.py", line 89 in _run
  File "/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/asyncio/base_events.py", line 2050 in _run_once
  File "/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/asyncio/base_events.py", line 683 in run_forever
  File "/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/asyncio/base_events.py", line 712 in run_until_complete
  File "/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/asyncio/runners.py", line 118 in run
  File "/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/asyncio/runners.py", line 195 in run
  File "/home/runner/work/qtile/qtile/libqtile/core/manager.py", line 224 in loop
  File "/home/runner/work/qtile/qtile/test/helpers.py", line 210 in run_qtile
  File "/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/multiprocessing/process.py", line 108 in run
  File "/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/multiprocessing/process.py", line 313 in _bootstrap
  File "/home/runner/work/qtile/qtile/.venv/lib/python3.13/site-packages/coverage/multiproc.py", line 51 in _bootstrap
  File "/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/multiprocessing/popen_fork.py", line 74 in _launch
  File "/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/multiprocessing/popen_fork.py", line 20 in __init__
  File "__init__", line ??? in __init__
  <invalid frame>

Extension modules: markupsafe._speedups, yaml._yaml, _cffi_backend, dbus_fast.signature, dbus_fast._private.marshaller, dbus_fast.message, dbus_fast.service, dbus_fast._private.address, dbus_fast.unpack, dbus_fast.message_bus, dbus_fast._private.unmarshaller, dbus_fast.aio.message_reader, gi._gi, cairo._cairo, gi._gi_cairo, psutil._psutil_linux, multidict._multidict, yarl._quoting_c, propcache._helpers_c, aiohttp._http_writer, aiohttp._http_parser, aiohttp._websocket.mask, aiohttp._websocket.reader_c, frozenlist._frozenlist (total: 24)
_ ERROR at setup of test_tasklist_click_task[1-wayland-tasklist_manager0-top-coords0] _

request = <SubRequest 'tasklist_manager' for <Function test_tasklist_click_task[1-wayland-tasklist_manager0-top-coords0]>>
manager_nospawn = <test.helpers.TestManager object at 0x7f348d06e3c0>
override_xdg = False
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f348f2a1c50>
position = 'top'

    @pytest.fixture
    def tasklist_manager(request, manager_nospawn, override_xdg, monkeypatch, position):
        monkeypatch.setattr("libqtile.widget.tasklist.has_xdg", override_xdg)
    
        config = getattr(request, "param", dict())
    
        class TasklistConfig(Config):
            auto_fullscreen = True
            groups = [
                libqtile.config.ScratchPad(
                    "SCRATCHPAD",
                    dropdowns=[
                        libqtile.config.DropDown("dd-a", spawn_cmd("dd-a"), on_focus_lost_hide=False),
                    ],
                ),
                libqtile.config.Group("a"),
                libqtile.config.Group("b"),
            ]
            layouts = [layout.Stack()]
            floating_layout = libqtile.resources.default_config.floating_layout
            keys = []
            mouse = []
            screens = [
                Screen(**{position: bar.Bar([TaskListTestWidget(name="tasklist", **config)], 28)})
            ]
    
>       manager_nospawn.start(TasklistConfig)

test/widgets/test_tasklist.py:69: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test/helpers.py:232: in start
    error = rpipe.recv()
            ^^^^^^^^^^^^
/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/multiprocessing/connection.py:250: in recv
    buf = self._recv_bytes()
          ^^^^^^^^^^^^^^^^^^
/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/multiprocessing/connection.py:430: in _recv_bytes
    buf = self._recv(4)
          ^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <multiprocessing.connection.Connection object at 0x7f348cf3df10>
size = 4, read = <built-in function read>

    def _recv(self, size, read=_read):
        buf = io.BytesIO()
        handle = self._handle
        remaining = size
        while remaining > 0:
            chunk = read(handle, remaining)
            n = len(chunk)
            if n == 0:
                if remaining == size:
>                   raise EOFError
E                   EOFError

/opt/hostedtoolcache/Python/3.13.12/x64/lib/python3.13/multiprocessing/connection.py:399: EOFError
---------------------------- Captured stdout setup -----------------------------
2026-03-27 13:52:35,492 libqtile core.py:log_cb():L105  [backend/backend.c:342] Loading user-specified backends due to WLR_BACKENDS: headless
2026-03-27 13:52:35,492 libqtile core.py:log_cb():L105  [backend/headless/backend.c:60] Creating headless backend
2026-03-27 13:52:35,492 libqtile core.py:log_cb():L105  [util/env.c:25] Loading WLR_RENDERER option: pixman
2026-03-27 13:52:35,492 libqtile core.py:log_cb():L105  [render/pixman/renderer.c:328] Creating pixman renderer
2026-03-27 13:52:35,493 libqtile core.py:log_cb():L105  [backend/headless/backend.c:17] Starting headless backend
2026-03-27 13:52:35,495 libqtile core.py:log_cb():L105  [server.c:190] Updating: 0,0 (800x600)
2026-03-27 13:52:35,501 libqtile core.py:log_cb():L105  [server.c:116] Running Wayland compositor on WAYLAND_DISPLAY=wayland-0
---------------------------- Captured stderr setup -----------------------------
Fatal Python error: Segmentation fault

Thread 0x00007f3495485b80 (most recent call first):
  File "/home/runner/work/qtile/qtile/libqtile/backend/wayland/core.py", line 221 in get_qtile_config_cb
  <invalid frame>

Extension modules: markupsafe._speedups, yaml._yaml, _cffi_backend, dbus_fast.signature, dbus_fast._private.marshaller, dbus_fast.message, dbus_fast.service, dbus_fast._private.address, dbus_fast.unpack, dbus_fast.message_bus, dbus_fast._private.unmarshaller, dbus_fast.aio.message_reader, gi._gi, cairo._cairo, gi._gi_cairo, psutil._psutil_linux, multidict._multidict, yarl._quoting_c, propcache._helpers_c, aiohttp._http_writer, aiohttp._http_parser, aiohttp._websocket.mask, aiohttp._websocket.reader_c, frozenlist._frozenlist (total: 24)
=========================== short test summary info ============================
ERROR test/widgets/test_tasklist.py::test_tasklist_skip_taskbar_defaults[1-wayland] - EOFError
ERROR test/widgets/test_tasklist.py::test_tasklist_click_task[1-wayland-tasklist_manager0-top-coords0] - EOFError
=========== 1307 passed, 61 skipped, 2 errors in 1075.47s (0:17:55) ============
make: *** [Makefile:31: check] Error 1
=== Listing coredumps ===
TIME                          PID  UID  GID SIG     COREFILE EXE                                                     SIZE
Fri 2026-03-27 13:52:32 UTC 30334 1001 1001 SIGSEGV present  /opt/hostedtoolcache/Python/3.13.12/x64/bin/python3.13 24.8M
Fri 2026-03-27 13:52:36 UTC 30436 1001 1001 SIGSEGV present  /opt/hostedtoolcache/Python/3.13.12/x64/bin/python3.13 25.0M
=== Stack trace for last coredump ===
           PID: 30436 (coverage)
           UID: 1001 (runner)
           GID: 1001 (runner)
        Signal: 11 (SEGV)
     Timestamp: Fri 2026-03-27 13:52:35 UTC (5min ago)
  Command Line: /home/runner/work/qtile/qtile/.venv/bin/python /home/runner/work/qtile/qtile/.venv/bin/coverage run -m pytest --backend=wayland
    Executable: /opt/hostedtoolcache/Python/3.13.12/x64/bin/python3.13
 Control Group: /system.slice/hosted-compute-agent.service
          Unit: hosted-compute-agent.service
         Slice: system.slice
       Boot ID: 91ae7b0e46fd44a9933e65170613bff2
    Machine ID: 29b09fb23a5a4dee8bc082d41f018839
      Hostname: runnervmrg6be
       Storage: /var/lib/systemd/coredump/core.coverage.1001.91ae7b0e46fd44a9933e65170613bff2.30436.1774619555000000.zst (present)
  Size on Disk: 25.0M
       Message: Process 30436 (coverage) of user 1001 dumped core.
                
                Module libzstd.so.1 from deb libzstd-1.5.5+dfsg2-2build1.1.amd64
                Module libsystemd.so.0 from deb systemd-255.4-1ubuntu8.14.amd64
                Module libstdc++.so.6 from deb gcc-14-14.2.0-4ubuntu2~24.04.1.amd64
                Module libudev.so.1 from deb systemd-255.4-1ubuntu8.14.amd64
                Module libgcc_s.so.1 from deb gcc-14-14.2.0-4ubuntu2~24.04.1.amd64
                Stack trace of thread 30437:
                #0  0x00007f3494a9eb2c pthread_kill (libc.so.6 + 0x9eb2c)
                #1  0x00007f3494a4527e raise (libc.so.6 + 0x4527e)
                #2  0x00007f3494a45330 n/a (libc.so.6 + 0x45330)
                #3  0x00007f3494a48806 getenv (libc.so.6 + 0x48806)
                #4  0x00007f3491132183 n/a (libexpat.so.1 + 0x20183)
                #5  0x00007f349111813b n/a (libexpat.so.1 + 0x613b)
                #6  0x00007f3491122061 n/a (libexpat.so.1 + 0x10061)
                #7  0x00007f34914576c9 n/a (libfontconfig.so.1 + 0x2f6c9)
                #8  0x00007f349145822d n/a (libfontconfig.so.1 + 0x3022d)
                #9  0x00007f3491458479 n/a (libfontconfig.so.1 + 0x30479)
                #10 0x00007f349145b437 n/a (libfontconfig.so.1 + 0x33437)
                #11 0x00007f349111c940 n/a (libexpat.so.1 + 0xa940)
                #12 0x00007f349111d773 n/a (libexpat.so.1 + 0xb773)
                #13 0x00007f349111e748 n/a (libexpat.so.1 + 0xc748)
                #14 0x00007f3491120f71 n/a (libexpat.so.1 + 0xef71)
                #15 0x00007f349111646f n/a (libexpat.so.1 + 0x446f)
                #16 0x00007f349112343e XML_ParseBuffer (libexpat.so.1 + 0x1143e)
                #17 0x00007f349145783e n/a (libfontconfig.so.1 + 0x2f83e)
                #18 0x00007f349145822d n/a (libfontconfig.so.1 + 0x3022d)
                #19 0x00007f349143f3b6 n/a (libfontconfig.so.1 + 0x173b6)
                #20 0x00007f349143a1c1 n/a (libfontconfig.so.1 + 0x121c1)
                #21 0x00007f349143f6bd FcInit (libfontconfig.so.1 + 0x176bd)
                #22 0x00007f3490a1f225 n/a (libpangoft2-1.0.so.0 + 0xc225)
                #23 0x00007f3490fc0e62 n/a (libglib-2.0.so.0 + 0x8be62)
                #24 0x00007f3494a9caa4 n/a (libc.so.6 + 0x9caa4)
                #25 0x00007f3494b29c6c n/a (libc.so.6 + 0x129c6c)
                
                Stack trace of thread 30436:
                #0  0x00007f3494aac208 n/a (libc.so.6 + 0xac208)
                #1  0x00007f3494aad714 malloc (libc.so.6 + 0xad714)
                #2  0x00007f348fa82aad xcursor_create_from_xcursor_images (libwlroots-0.19.so + 0xc4aad)
                #3  0x00007f348fa82c00 load_callback (libwlroots-0.19.so + 0xc4c00)
                #4  0x00007f348fa84752 load_all_cursors_from_dir (libwlroots-0.19.so + 0xc6752)
                #5  0x00007f348fa8490e xcursor_load_theme_protected (libwlroots-0.19.so + 0xc690e)
                #6  0x00007f348fa849e1 xcursor_load_theme_protected (libwlroots-0.19.so + 0xc69e1)
                #7  0x00007f348fa84a62 xcursor_load_theme (libwlroots-0.19.so + 0xc6a62)
                #8  0x00007f348fa82d52 wlr_xcursor_theme_load (libwlroots-0.19.so + 0xc4d52)
                #9  0x00007f348fa7792f wlr_xcursor_manager_load (libwlroots-0.19.so + 0xb992f)
                #10 0x00007f348fb08236 n/a (/home/runner/work/qtile/qtile/libqtile/backend/wayland/_ffi.cpython-313-x86_64-linux-gnu.so + 0x1f236)
                ELF object binary architecture: AMD x86-64


warning: Can't open file /dev/shm/wlroots-KIOmgO (deleted) during file-backed mapping note processing

warning: Can't open file /dev/shm/wlroots-GHiboO (deleted) during file-backed mapping note processing

warning: Can't open file /dev/shm/pym-16853-c1v7w9b5 (deleted) during file-backed mapping note processing
[New LWP 30437]
[New LWP 30436]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Core was generated by `/home/runner/work/qtile/qtile/.venv/bin/python /home/runner/work/qtile/qtile/.v'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007f3494a9eb2c in pthread_kill () from /lib/x86_64-linux-gnu/libc.so.6
[Current thread is 1 (Thread 0x7f348cc386c0 (LWP 30437))]
#0  0x00007f3494a9eb2c in pthread_kill () at /lib/x86_64-linux-gnu/libc.so.6
#1  0x00007f3494a4527e in raise () at /lib/x86_64-linux-gnu/libc.so.6
#2  0x00007f3494a45330 in <signal handler called> () at /lib/x86_64-linux-gnu/libc.so.6
#3  0x00007f3494a48806 in getenv () at /lib/x86_64-linux-gnu/libc.so.6
#4  0x00007f3491132183 in ??? () at /lib/x86_64-linux-gnu/libexpat.so.1
#5  0x00007f349111813b in ??? () at /lib/x86_64-linux-gnu/libexpat.so.1
#6  0x00007f3491122061 in ??? () at /lib/x86_64-linux-gnu/libexpat.so.1
#7  0x00007f34914576c9 in ??? () at /lib/x86_64-linux-gnu/libfontconfig.so.1
#8  0x00007f349145822d in ??? () at /lib/x86_64-linux-gnu/libfontconfig.so.1
#9  0x00007f3491458479 in ??? () at /lib/x86_64-linux-gnu/libfontconfig.so.1
#10 0x00007f349145b437 in ??? () at /lib/x86_64-linux-gnu/libfontconfig.so.1
#11 0x00007f349111c940 in ??? () at /lib/x86_64-linux-gnu/libexpat.so.1
#12 0x00007f349111d773 in ??? () at /lib/x86_64-linux-gnu/libexpat.so.1
#13 0x00007f349111e748 in ??? () at /lib/x86_64-linux-gnu/libexpat.so.1
#14 0x00007f3491120f71 in ??? () at /lib/x86_64-linux-gnu/libexpat.so.1
#15 0x00007f349111646f in ??? () at /lib/x86_64-linux-gnu/libexpat.so.1
#16 0x00007f349112343e in XML_ParseBuffer () at /lib/x86_64-linux-gnu/libexpat.so.1
#17 0x00007f349145783e in ??? () at /lib/x86_64-linux-gnu/libfontconfig.so.1
#18 0x00007f349145822d in ??? () at /lib/x86_64-linux-gnu/libfontconfig.so.1
#19 0x00007f349143f3b6 in ??? () at /lib/x86_64-linux-gnu/libfontconfig.so.1
#20 0x00007f349143a1c1 in ??? () at /lib/x86_64-linux-gnu/libfontconfig.so.1
#21 0x00007f349143f6bd in FcInit () at /lib/x86_64-linux-gnu/libfontconfig.so.1
#22 0x00007f3490a1f225 in ??? () at /lib/x86_64-linux-gnu/libpangoft2-1.0.so.0
#23 0x00007f3490fc0e62 in ??? () at /lib/x86_64-linux-gnu/libglib-2.0.so.0
#24 0x00007f3494a9caa4 in ??? () at /lib/x86_64-linux-gnu/libc.so.6
#25 0x00007f3494b29c6c in ??? () at /lib/x86_64-linux-gnu/libc.so.6
Error: Process completed with exit code 2.
```

https://github.com/qtile/qtile/actions/runs/23648899481/job/68888140869

--- a/libqtile/__init__.py
+++ b/libqtile/__init__.py
@@ -25,6 +25,7 @@ class DynamicLibraries(enum.Enum):
     PANGO = "pango-1.0"
     PANGOCAIRO = "pangocairo-1.0"
     XCBCURSOR = "xcb-cursor"
+    FONTCONFIG = "fontconfig"
 
 
 def find_library(key: DynamicLibraries) -> str | None:

--- a/libqtile/pango_ffi.py
+++ b/libqtile/pango_ffi.py
@@ -37,6 +37,9 @@ pango_ffi.cdef(
     typedef ... GError;
     typedef int gint;
 
+    int
+    FcInit(void);
+
     void
     pango_cairo_show_layout (cairo_t *cr,
                              PangoLayout *layout);

--- a/libqtile/pangocffi.py
+++ b/libqtile/pangocffi.py
@@ -4,6 +4,11 @@ from libqtile.pango_ffi import pango_ffi as ffi
 gobject = ffi.dlopen(find_library(DynamicLibraries.GOBJECT))  # type: ignore
 pango = ffi.dlopen(find_library(DynamicLibraries.PANGO))  # type: ignore
 pangocairo = ffi.dlopen(find_library(DynamicLibraries.PANGOCAIRO))  # type: ignore
+fontconfig = ffi.dlopen(find_library(DynamicLibraries.FONTCONFIG))  # type: ignore
+
+
+def init_fontconfig():
+    fontconfig.FcInit()
 
 
 def create_layout(cairo_t):

--- a/libqtile/scripts/start.py
+++ b/libqtile/scripts/start.py
@@ -7,7 +7,7 @@ from sys import exit
 from typing import TYPE_CHECKING
 
 import libqtile.backend
-from libqtile import confreader, qtile
+from libqtile import confreader, pangocffi, qtile
 from libqtile.log_utils import logger
 from libqtile.utils import VERSION, get_config_file
 
@@ -84,6 +84,9 @@ def start(options):
         locale.setlocale(locale.LC_ALL, "")
     except locale.Error:
         pass
+
+    # Initialise fontconfig before starting qtile to prevent races
+    pangocffi.init_fontconfig()
 
     libpath = (LIBQTILE_PATH / "..").resolve()
     logger.warning(f"Starting Qtile {VERSION} from {libpath}")

--- a/test/helpers.py
+++ b/test/helpers.py
@@ -19,7 +19,7 @@ import traceback
 from abc import ABCMeta, abstractmethod
 from pathlib import Path
 
-from libqtile import command, config, ipc, layout
+from libqtile import command, config, ipc, layout, pangocffi
 from libqtile.confreader import Config
 from libqtile.core.manager import Qtile
 from libqtile.lazy import lazy
@@ -191,6 +191,8 @@ class TestManager:
                 os.environ.pop("DISPLAY", None)
                 os.environ.pop("WAYLAND_DISPLAY", None)
                 init_log(self.log_level)
+                # Initialise fontconfig before starting qtile to prevent races
+                pangocffi.init_fontconfig()
                 kore = self.backend.create()
                 os.environ.update(self.backend.env)
                 from libqtile.core.lifecycle import lifecycle

